server/methods/core/groups.js

```diff
@@ -124,14 +128,19 @@ Meteor.methods({
     const group = Groups.findOne({ _id: groupId }) || {};
     const { permissions, shopId, slug } = group;
     const loggedInUserId = Meteor.userId();
-    const canInvite = Reaction.canInviteToGroup({ group, user: Meteor.user() });
+    const canInvite = Reaction.canInviteToGroup({ group });
 
-    // Owners can invite to any group.
-    if (!Reaction.hasPermission("owner", loggedInUserId, shopId)) {
-      // Admins can invite to only groups they belong
-      if (!canInvite) {
-        throw new Meteor.Error(403, "Access Denied");
-      }
+    // we are limiting group method actions to only users with admin roles
+    // this also include shop owners, since they have the `admin` role in their Roles.GLOBAL_GROUP
+    if (!Reaction.hasPermission("admin", loggedInUserId, shopId)) {
+      throw new Meteor.Error(403, "Access Denied");
+    }
+
+    // Users with `owner` and/or `admin` roles can invite to any group
+    // Also a user with `admin` can invite to only groups they have permissions that are a superset of
```

Comment:
These two comment lines seem to contradict each other.
Can a user with `admin` permissions invite to any group, or only groups they have superset of permissions for?

Reply:
@impactmass want to make sure you see this comment as I think these comments should be clarified, but going to merge anyway.

Reply:
I'll note to update the comment in next PR I open, but I meant the latter: a user with admin can invite to only groups they have superset of permissions for.

Reply:
That's what I figured. Next PR is good